backend/app/config.py

```diff
@@ -22,6 +22,7 @@ class Settings(BaseSettings):
 
     # Misc streamchaser settings
     environment: Environment = Environment.DEVELOPMENT
+    fill_cache_on_starup: bool = True
```

Comment:
Minor spelling mistake, starup -> startup

Reply:
Done

Reply:
Did this get reintroduced during the rebase?

Reply:
Yup, I messed up big time rebasing :D I've deleted the setting now